hack/lib/golang.sh

```diff
@@ -496,9 +496,6 @@ EOF
 # ${PATH}. It will also check that the Go version is good enough for the
 # Kubernetes build.
 #
-# Inputs:
-#   KUBE_EXTRA_GOPATH - If set, this is included in created GOPATH
```

Comment:
noting that the only usage of this organization wide appears to be fixed in this PR (looking beyond this repo) https://cs.k8s.io/?q=KUBE_EXTRA_GOPATH&i=nope&files=&excludeFiles=&repos=

Reply:
I am working on a comprehensive PR to adopt Go workspaces - I'll ask that we refrain for touching this logic too much - it will just make my work harder.  This specific case is OK :)